ingestion_program/systematics.py

```diff
@@ -553,6 +567,7 @@ def get_bootstraped_dataset(
     ttbar_scale=None,
     diboson_scale=None,
     bkg_scale=None,
+    poisson = True
 ):
     """
     Generate a bootstraped dataset
```

Comment:
bootstraped = > bootstrapped (here and everywhere)

Reply:
I fixed it